R/cut.R

```diff
@@ -19,6 +19,17 @@
 #'  `step_cut()` will call `base::cut()` in the baking step with
 #'  `include.lowest` set to `TRUE`.
 #'
+#' # Tidying
+#'
+#' When you [`tidy()`][tidy.recipe()] this step, a tibble returned with 3
+#' columns `terms`, `value` and `id`:
+#'
+#' \describe{
+#'   \item{terms}{character, the selectors or variables selected}
+#'   \item{value}{character, the location of the cuts}
```

Comment:
you would think this should be numeric. but it results characters. related a bit to https://github.com/tidymodels/recipes/issues/1229